public/index.html

```diff
@@ -57,7 +57,36 @@ <h2>What is it?</h2>
           <br/>
           <br/>
           There's no agenda, no presentations, no requirements, and hackers and
-          the hacker-curious of all levels are welcome to attend.
+          the hacker-curious of all levels are welcome to attend. However, we
+          do partner with other tech groups every week to expose attendees to a
+          variety of tech knowledge in the city. 
+          <br/>
+          <br/>
+          Check out our partners as they present on a monthly rotation in
+          <a href="http://threekeysnola.com/">Ace Hotel's 3 Keys Room!</a>
+          <br/>
+          <table>
```

Comment:
Minor visual detail, could you wrap this table in a `<center>` element or equivalent? It's left-aligned and looks a little wacky.

Reply:
Done. I also centered the table headers to make them more prominent.